README.md

```diff
@@ -1 +1,21 @@
-# Create React Admin Amplify App
\ No newline at end of file
+# Create React Admin Amplify App
```

Comment:
Nice! This overall algorithm is exactly how I envisioned it

Reply:
Oh! I think it should provision a user and in the output of the `create-react-admin-amplify-app` command it should say in all capitals:

> LOGIN WITH THE FOLLOWING CREDENTIALS:
> username: <username>
> password: <password>

Sounds a little silly but I got stuck here in a few different ways. You can have a completely working app but if it requires a login and you're not already familiar it's like.... how do i login for the first time? is there a default admin account created?

Then I got into the incognito admin console and figured out how to create a user here but then I'm having questions about confirming the user, and the login still didn't work and I had no idea if i was actually on the right path towards actually logging in, which was pretty frustrating given how simple and core it is.

Reply:
A later revision could even include a command to generate additional admin users...

Reply:
Yep that's a nice API to have. Amplify itself uses some steps in the AWS console so we could have something akin to that initially. I don't know how hard it would be to automate the whole thing though. looks like a security/convenience trade-off

Reply:
The security trade-off is because we're generating a user account or because we display the password in plaintext in the browser? Or both?

Reply:
For me it was a big risk to attempt to set the auth up without experience in these tools so I saw it as an opportunity to bake best practices in.

If the password is randomly generated I think this is fine personally? Open to alternate perspective but devs copy/paste SSH keys and AWS creds so I hadn't seen this as additional risk.

If we wanted to kick it up a notch we could require the password be changed after login or just send an invite to dev's email?

Alternately.... if the registration page is enabled by default this solves the problem.

Reply:
both